src/Main.java

```diff
@@ -1,54 +1,51 @@
+import java.time.LocalDate;
+
 public class Main {
-    public static String calculateLeapYear(int year) {
+    public static void calculateLeapYear(int year) {
```

Comment:
метод ничего не считает, он выводит
```suggestion
    public static void printLeapYear(int year) {
```